src/helpers/get-survey-answers.ts

```diff
@@ -0,0 +1,33 @@
+import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import {parseISO} from "date-fns";
+
+export default async function(props : SurveyAnswersQuery) : Promise<SurveyAnswer[]> {
```

Comment:
I would export this function as "queryAllSurveyAnswers"...that is what distinguishes it from the SDK querySurveyAnswers method...it is getting *all* the answers